.dockerignore

```diff
@@ -11,7 +11,6 @@
 **/fxa-content-server-l10n
 packages/*/build
 packages/fxa-auth-server/.mail_output
-!packages/fxa-auth-server/docs/pushpayloads.schema.json
```

Comment:
I took the liberty to move this one-off json file that is used by auth-server into its lib folder since it isn't just documentation